pgx/src/trigger_support/pg_trigger.rs

```diff
@@ -247,27 +269,4 @@ impl PgTrigger {
     pub fn fcinfo(&self) -> &pg_sys::FunctionCallInfo {
         self.fcinfo.borrow()
     }
-
-    /// Eagerly evaluate the data in this `PgTrigger` and build a safely accessible structure
-    /// which mimics the data provided to a PL/pgSQL trigger.
-    pub unsafe fn to_safe(&self) -> Result<PgTriggerSafe, PgTriggerError> {
```

Comment:
It's not immediately obvious: do we lose the "lazy evaluation" quality here? This seems fine otherwise.

Reply:
Since `PgTriggerSafe` is gone users will only use `PgTrigger` and that struct *is* lazy.  It does next to nothing during its `from_fcinfo()` function, which is how it's created.